### DISCUSSION TITLE
Mermaid?

### DISCUSSION THREAD

**Discussion by Cammagno** [SkepticMystic/breadcrumbs#393] [Q&A]
3.0.0 changelog states: "Working Mermaid diagrams to visualise the implied relations"
so...
is it possible to use breadcrumbs to generate Mermaid at all?
Where can I find some info about how this works?

If not, can someone explain me what the meaning of that line, plese? :)

EDIT: got it, it was about the diagrams in the plugin settings.

**Comment by michaelpporter** [comment]
Closing — v4 ships a Mermaid codeblock (`type: mermaid`). Reopen if you need more than that.
